inc/Widget/SmailyWidget.php

```diff
@@ -155,46 +155,252 @@ public function form( $instance ) {
-			if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) {
-				echo '<option value="">' . esc_html__( '-No Autoresponder-', 'smaily' ) . ' </option>';
-				foreach ( $autoresponder_list as $autoresponder ) {
-					echo '<option value="' . htmlentities( json_encode( $autoresponder ) ) . '">' . $autoresponder['name'] . '</option>';
-				}
-			}
-			?>
-		</select>
-		<?php endif; ?>
-		<?php
 
+		<div class="smaily">
+			<!-- Title. -->
+			<div class="section">
+				<h2>
+					<?php echo esc_html_e( 'Title', 'smaily' ); ?>
+				</h2>
+				<input 
+					class="widefat" 
+					id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>" 
+					name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>" 
+					type="text" value="<?php echo esc_attr( $title ); ?>">
+			</div>
+
+			<!-- Autoresponder. -->
+			<?php if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) : ?>
+			<div class="section">
+				<h2>
+					<?php echo esc_html_e( 'Autoresponder', 'smaily' ); ?>
+				</h2>
+				<select 
+					id="<?php echo esc_attr( $this->get_field_id( 'autoresponder' ) ); ?>" 
+					name="<?php echo esc_attr( $this->get_field_name( 'autoresponder' ) ); ?>" 
+					class="widefat" 
+					style="width:100%;">
+					<option value=""><?php echo esc_html_e( '-No Autoresponder-', 'smaily' ); ?></option>
+					<?php if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) : ?>
+					<?php foreach ( $autoresponder_list as $autoresponder ) : ?>
+						<option value="<?php echo htmlentities( json_encode( $autoresponder ) ); ?>"<?php if ( $autoresponder['id'] == $current_autoresponder['id'] ) : ?> selected<?php endif; ?>><?php echo $autoresponder['name']; ?></option>
+					<?php endforeach; ?>
+					<?php endif; ?>
+				</select>
+				<p>
+					<?php echo esc_html_e( 'Select a suitable form-submitted trigger automation workflow from the list or create a new autoresponder on your Smaily account to be added to the list.', 'smaily' ); ?>
+				</p>
+			</div>
+			<?php endif; ?>
+
+			<!-- Layouts. -->
+			<div class="section">
+				<h2>
+					<?php echo esc_html_e( 'Layout', 'smaily' ); ?>
+				</h2>
+				<p>
+					<?php echo esc_html_e( 'Choose a preferred layout.', 'smaily' ); ?>
+				</p>
+
+				<div class="smaily-layout-container">
+					<p> <!-- Select layout. -->
+						<input 
+							class="radio" 
+							id="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>" 
+							name="<?php echo $this->get_field_name('form_layout'); ?>"
+							type="radio" value="smaily-layout-1" 
+							<?php if ( $form_layout === 'smaily-layout-1' ) { echo 'checked="checked"'; } ?> />
+						<label for="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>">
+						<strong><?php echo esc_html_e( 'Layout 1', 'smaily' ); ?></strong> &minus; <?php echo esc_html_e( 'email address with spaced button', 'smaily' ); ?></label>
```

Comment:
Odd indentation for layout labels.

I suggest you change it to something like:
```suggestion
						<label for="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>">
							<strong><?php echo esc_html_e( 'Layout 1', 'smaily' ); ?></strong> &minus; <?php echo esc_html_e( 'email address with spaced button', 'smaily' ); ?>
						</label>
```